src/main/java/org/manifold/compiler/front/Main.java

```diff
@@ -129,172 +118,33 @@ public Schematic invokeFrontend(CommandLine cmd) throws Exception {
       }
     }
 
-    TypeChecker.typecheck(namespaces, defaultNamespace);
+    log.debug("top-level identifiers:");
+    for (VariableIdentifier id :
+        defaultNamespace.getPrivateScope().getSymbolIdentifiers()) {
+      log.debug(id);
+    }
 
-    ExpressionGraph exprGraph = new ExpressionGraph(
-        defaultNamespace.getPrivateScope());
-    exprGraph.buildFrom(expressions);
-    exprGraph.removeUnconnectedEdges();
-    exprGraph.optimizeOutVariables();
+    // let's see if we can do this before static type-checking
```

Comment:
Actually it was more of a "note to self" before the static typechecker was removed. Now that it doesn't exist any more, this is kind of meaningless.